src/libs/vpatterndb/vpiece.h

```diff
@@ -72,102 +72,109 @@ class VPointF;
+    QPainterPath             getNotchesPath(const VContainer *data,
+                                           const QVector<QPointF> &seamAllowance = QVector<QPointF>()) const;
 
-    bool IsInLayout() const;
-    void SetInLayout(bool inLayout);
+    bool                     IsInLayout() const;
+    void                     SetInLayout(bool inLayout);
 
-    bool IsUnited() const;
-    void SetUnited(bool united);
+    bool                     IsUnited() const;
+    void                     SetUnited(bool united);
 
-    QString GetFormulaSAWidth() const;
-    void    SetFormulaSAWidth(const QString &formula, qreal value);
+    QString                  GetFormulaSAWidth() const;
+    void                     SetFormulaSAWidth(const QString &formula, qreal value);
 
-    QVector<quint32> GetInternalPaths() const;
-    QVector<quint32> &GetInternalPaths();
-    void             SetInternalPaths(const QVector<quint32> &iPaths);
+    QVector<quint32>         GetInternalPaths() const;
+    QVector<quint32>        &GetInternalPaths();
+    void                     SetInternalPaths(const QVector<quint32> &iPaths);
 
-    QVector<CustomSARecord> GetCustomSARecords() const;
-    QVector<CustomSARecord> &GetCustomSARecords();
-    void                    SetCustomSARecords(const QVector<CustomSARecord> &records);
+    QVector<CustomSARecord>  GetCustomSARecords() const;
+    QVector<CustomSARecord>  &GetCustomSARecords();
+    void                     SetCustomSARecords(const QVector<CustomSARecord> &records);
 
-    QVector<quint32> GetPins() const;
-    QVector<quint32> &GetPins();
-    void             SetPins(const QVector<quint32> &pins);
+    QVector<quint32>         GetPins() const;
+    QVector<quint32>        &GetPins();
+    void                     SetPins(const QVector<quint32> &pins);
 
-    QVector<quint32> MissingNodes(const VPiece &det) const;
-    QVector<quint32> MissingCSAPath(const VPiece &det) const;
-    QVector<quint32> MissingInternalPaths(const VPiece &det) const;
-    QVector<quint32> MissingPins(const VPiece &det) const;
+    QVector<quint32>         MissingNodes(const VPiece &det) const;
+    QVector<quint32>         MissingCSAPath(const VPiece &det) const;
+    QVector<quint32>         MissingInternalPaths(const VPiece &det) const;
+    QVector<quint32>         MissingPins(const VPiece &det) const;
 
-    void                   SetPatternPieceData(const VPieceLabelData &data);
-    VPieceLabelData&       GetPatternPieceData();
-    const VPieceLabelData& GetPatternPieceData() const;
+    void                     SetPatternPieceData(const VPieceLabelData &data);
+    VPieceLabelData&         GetPatternPieceData();
+    const VPieceLabelData&   GetPatternPieceData() const;
 
     void                     SetPatternInfo(const VPatternLabelData &info);
     VPatternLabelData&       GetPatternInfo();
-    const VPatternLabelData& GetPatternInfo() const;
+    const VPatternLabelData &GetPatternInfo() const;
```

Comment:
&

Reply:
This is one of those "style" conventions that is inconsistent  throughout the code. 
White space is irrelevant to the compiler. so 

VPatternLabelData& GetPatternInfo() is the same as VPatternLabelData &GetPatternInfo()
Qt use the former... and Most of the time Roman used that , but not always. In fact there are time where he was inconsistent even on the same line. 

I prefer the former. 

In either case I would like to open a discussion on the forums about creating a code style standard, so we can decide what style should be used and where... then stick to that. I just have a habit of cleaning code up when it in front of me. :)

Reply:
ok more super
Put thoughts about code style here on wiki.